tests/unit/scripts/test_ask_qdrant.py

```
@@ -0,0 +1,143 @@
+from unittest.mock import MagicMock, Mock, PropertyMock, mock_open, patch
+
+class TestMainFunction:
+    @patch('builtins.print')
+    def test_main_no_matches_from_qdrant(
+        self,
+        _mock_print,
+        _mock_ensure_env,
+        mock_load_settings,
```

Ruff (0.13.1)

Unused method argument: `mock_load_settings`

(ARG002)